lib/file-operations.js

```diff
@@ -73,12 +73,12 @@ function getModeDiff(fsMode, vinylMode) {
 
 function getTimesDiff(fsStat, vinylStat) {
 
-  var mtime = date(vinylStat.mtime);
+  var mtime = date(vinylStat.mtime) || null;
```

Comment:
Why did this happen?

Reply:
Because value-or-function now returns `undefined` for invalid input, and the expression below this line is expecting `null`: Oddly `+null` yields `0`, whereas `+undefined` yields `NaN`, for some reason.

Reply:
Maybe we should make them 0 then?

Reply:
Yeah, that would work. But now that I look at this again, the expressions with `+` seem slightly wrong here. Aren't we discounting `0` as a valid date (namely 1970-01-01 00:00:00)? Unlikely to ever come up, I guess, but still.

Anyway, I've amended to `0` for now.